OBJECTIONS.md

```diff
@@ -4,87 +4,87 @@
+- The _Controller_ exists on the client browser. The browser receives button clicks, mouse movements, and key presses, just like in the GUI pattern. The client browser (as _Controller_) sends an HTTP request event to the server when the user clicks a link, or a submits a form, etc.
 
-- The _Model_ exists on the server. When the client browser (as _Controller_) sends an HTTP Request "event" to the server, the client receives back the updated _Model_ data as an HTTP Response "event". (The data may be an entire page or only a portion thereof.) This is like the GUI pattern as well, except that the client is interacting with remote, instead of local, resources encapsulating business logic.
+- The _Model_ exists on the server. When the client browser (as _Controller_) sends an HTTP request event to the server, the client receives back the updated _Model_ data as an HTTP response event. (The data may be an entire page or only a portion thereof.) This is like the GUI pattern as well, except the client is interacting with remote, instead of local, resources encapsulating business logic.
 
 - The _View_ exists on the client browser. The client browser (as _View_), receiving the updated _Model_ data in the form of an HTTP Response, re-renders its screen with the changes, either as an entirely new page or as changes to an existing page.
 
-While this might be a reasonable way to describe the user interface of the **client** side of the network interaction, it says nothing at all about the user interface on the **server** side of that interaction.
+While this might be a reasonable way to describe the user interface of the **client** side of the network interaction, it says nothing about the user interface on the **server** side of that interaction.
 
-_Action Domain Responder_ addresses specifically the user interface components and collaborations that exist on the **server** side.
+_Action Domain Responder_ specifically addresses the user interface components and collaborations existing on the **server** side.
 
-> N.b.: One correspondent suggests there is "an interesting proof that something is up with MVC in a web context" to be found when there is no browser involved, such as using `curl` or `wget` to interact with a server-side application that emits JSON response bodies. "The HTML message body is no more a user interface than a JSON message body -- both are simply serialization formats" as far as the server is concerned.
+> N.b.: One correspondent suggests there is "an interesting proof that something is up with MVC in a web context" to be found when there is no browser involved, such as using `curl` or `wget` to interact with a server-side application that emits JSON response bodies. "The HTML message body is no more a user interface than a JSON message body---both are simply serialization formats" as far as the server is concerned.
```

Comment:
I don't think a triple dash is ever considered correct in lieu of proper en or em dashes. A double dash would do, with or without surrounding spaces.